glib/src/gobject/binding_group.rs

```diff
@@ -111,7 +111,7 @@ impl<'a> BindingGroupBuilder<'a> {
             user_data: ffi::gpointer,
         ) -> ffi::gboolean {
             let transform_data =
-                &*(user_data as *const (TransformFn, TransformFn, ParamSpec, ParamSpec));
```

Comment:
It would be good to have a test with the transform functions to make sure this actually works properly.

Reply:
@A6GibKm This is still pending

Reply:
Added tests but they fail here locally with 
```
Caused by:
  process didn't exit successfully: `/var/home/user/Projects/gtk-rs-core/target/debug/deps/glib-73fbdc0a479b0288` (signal: 11, SIGSEGV: invalid memory reference)
  ```

Reply:
Good that we added a test then :)

Reply:
I think the crash is because this should be `String, ParamSpec`?

Reply:
It is like that now

Reply:
Sorry not this one, it's in another spot. See my other review